### ISSUE TITLE
Abell 21 RA/Dec Catalog Error

### ISSUE THREAD

**Issue by eseavey7** [Stellarium/stellarium#1473]
I have noticed this error since 0.20.2 however, it may have appeared in earlier versions, and is still present in 0.20.4.
When performing a find for Abell 21, the incorrect RA/Dec are given, and Stellarium centers in the incorrect direction.  Abell 21 is close to the NGC 2395 cluster and the Abell 21 Nebula image, however, does have the correct RA/Dec.

**Comment by alex-w** [comment]
Abell 21 in Stellarium is galaxy cluster - not nebula!

**Comment by alex-w** [comment]
See http://simbad.u-strasbg.fr/simbad/sim-basic?Ident=Abell+21&submit=SIMBAD+search

**Comment by eseavey7** [comment]
My apologies, I see now that Abell in Stellarium is consistently referring to the ACO catalog.  Has the Abell catalog of planetary nebulae been removed?

**Comment by alex-w** [comment]
Yes, it was removed

**Comment by silas1037** [comment]
@alex-w  what about recovering abell pn, just append their names into names.dat?

**Comment by gzotti** [comment]
@alex-w one could be called Abell, the other ACO.

**Comment by alex-w** [comment]
The Abell’s planetary catalog was removed by 2 reasons: 1) all 6 different Abell’s catalogs has same designation prefix - Abell - and this is nightmare 2) professionals ask me why planetary catalog has Abell designation and suggested me that catalog of galaxies groups is more familiar and frequently used. 

SIMBAD use Abell and ACO designations for gal. groups and PN A66 for catalog of planetary nebulae which we used in Stellarium.
